File: snippets/fsharp/System/String/PadRight/source1.fs
module source1

// <Snippet1>
let str = "forty-two"
let pad = '.'

printfn $"{str.PadRight(15, pad)}"    // Displays "forty-two......".
printfn $"{str.PadRight(2, pad)}"    // Displays "forty-two".
// </Snippet1>

File: snippets/csharp/System/String/PadRight/source1.cs
using System;

public class PadRightCharacterSample
{
    public static void Run()
    {
        // <Snippet1>
        string str = "forty-two";
        char pad = '.';

        Console.WriteLine(str.PadRight(15, pad));    // Displays "forty-two......".
        Console.WriteLine(str.PadRight(2, pad));    // Displays "forty-two".
                                                    // </Snippet1>
    }
}
